Lib/unittest/test/test_case.py

```diff
@@ -799,8 +801,9 @@ def testAssertSequenceEqualMaxDiff(self):
         self.assertEqual(self.maxDiff, 80*8)
         seq1 = 'a' + 'x' * 80**2
         seq2 = 'b' + 'x' * 80**2
-        diff = '\n'.join(difflib.ndiff(pprint.pformat(seq1).splitlines(),
-                                       pprint.pformat(seq2).splitlines()))
+        diff = unittest.case._heuristic_diff(pprint.pformat(seq1).splitlines(),  # type: ignore
```

Comment:
```suggestion
        diff = unittest.case._heuristic_diff(pprint.pformat(seq1).splitlines(),
```
There's no type checker to ignore here. This reappears several times below.